reactioncommerce/reaction
# Fix order table RTL alignment

Resolves #2818. Also, adds translations for order table headers.

### How to test
- Change language to an RTL one 
- Visit the orders dashboard
- Observe that the table and its elements are properly aligned

P.S. `reaction reset` for the translations on the order table headers to take effect.
